spec/fixtures/mtr.yaml

```diff
@@ -0,0 +1,31 @@
+type: MeshTCPRoute
+name: tcp-route-1
+mesh: default
+spec:
+  targetRef:
+    kind: MeshSubset
+    tags:
+      app: frontend
+  to:
+    - targetRef:
+        kind: MeshService
+        name: backend
+        namespace: kuma-demo
+        _port: 3001
+      rules:
+        - default:
+            backendRefs:
+              - kind: MeshService
+                name: backend
+                _version: v1
+                namespace: kuma-demo
+                _port: 3001
+                port: 3001
```

Comment:
do we need `port` in this example?

Reply:
we only need `port`, not `_port`

Reply:
so we can remove it from this example

Reply:
so we now don't use _port in our examples? won't it break all of the rest already migrated policies?

Reply:
`_port` in `targetRef` is different than `port` in `backendRef`